res/controllers/Pioneer-DDJ-200-scripts.js

```diff
@@ -1,406 +1,594 @@
+    Jog (Platter)   rotate      Vinyl mode on   Bn  22      hh      -           result = value - 64
+                                Vinyl mode off  Bn  23      hh      -           result = value - 64
+                                shift           Bn  29      hh      -           result = value - 64
+    Jog (Platter)   touch                       9n  36      hh      -           OFF=0x00, ON=0x7F
+                                shift           9n  67      hh      -           OFF=0x00, ON=0x7F
+    Jog (Side)      rotate      normal/shift    Bn  21      hh      -           result = value - 64
+    Beat Sync       press                       9n  58      hh  [midi in]       OFF=0x00, ON=0x7F
+                    long press                  9n  5C      hh      -           OFF=0x00, ON=0x7F
+                    press       shift           9n  60      hh  [midi in]       OFF=0x00, ON=0x7F
+    Tempo           slide       normal/shift    Bn  00/20   xx      -           MSB / LSB
+    Play/Pause      press                       9n  0B  hh      [midi in]       OFF=0x00, ON=0x7F
+                    press       shift           9n  47  hh      [midi in]       OFF=0x00, ON=0x7F
+    Cue             press                       9n  0C  hh      [midi in]       OFF=0x00, ON=0x7F
+                    press       shift           9n  48  hh      [midi in]       OFF=0x00, ON=0x7F
+    Shift           press                       9n  3F  hh          -           OFF=0x00, ON=0x7F
+    EQ Hi           rotate                      Bn  07/27   xx      -           MSB / LSB
+    EQ Mid          rotate                      Bn  0B/2B   xx      -           MSB / LSB
+    EQ Low          rotate                      Bn  0F/2F   xx      -           MSB / LSB
+    Color Fx CH1    rotate                      B6  17/37   xx      -           MSB / LSB
+    Color Fx CH2    rotate                      B6  18/38   xx      -           MSB / LSB
+    Master Cue      press                       96  63      hh  [midi in]       OFF=0x00, ON=0x7F
+                    press       shift           96  78      hh  [midi in]       OFF=0x00, ON=0x7F
+    Cue (Headpho)   press                       9n  54      hh  [midi in]       OFF=0x00, ON=0x7F
+                    press       shift           9n  68      hh  [midi in]       OFF=0x00, ON=0x7F
+    Ch Fader        slide                       Bn  13/33   xx      -           MSB / LSB
+    Crossfader      slide                       B6  1F/3F   xx      -           MSB / LSB
+    both above      zero->notzero   shift       9n  66  hh      [midi in]       OFF=0x00, ON=0x7F (PLAY message for fader start)
+                    notzero->zero   shift       9n  52  hh      [midi in]       OFF=0x00, ON=0x7F (CUE message for fader start)
+    TransitionFX    press                       96  59  hh      [midi in]       OFF=0x00, ON=0x7F
+                    press           shift       96  5A  hh      [midi in]       OFF=0x00, ON=0x7F
+
```

Comment:
I just went through the DDJ200 manual and also found that there were dedicated "loaded" LEDs you didn't document or seem to have used `0x9F, i, 0x7F` (where `i` is the channel). Would you mind documenting that and maybe also using in the mapping. Simplify create a plain `Component` and define its `outKey` and no `inKey`.